kerlescan/config.py

```diff
@@ -30,6 +32,15 @@ def load_hosts_setting(env_name, clowder_endpoint, default):
     return os.getenv(env_name, default)
 
 
+def load_setting(env_name, clowder_key, default):
+    if isClowderEnabled():
+        return getattr(LoadedConfig, clowder_key, None)
+
+    return os.getenv(env_name, default)
+
+
+tls_ca_path = load_setting("TLS_CA_PATH", "tlsCAPath", None)
```

Comment:
Default value should be `True`. https://requests.readthedocs.io/en/latest/api/#requests.request

Reply:
@vkrizan This needs to have a falsey value by default because apps need to be able to configure themselves differently if it is not set.

i.e.
``` python
if tls_ca_path:
  # use https
else:
    # use http
```

Also of note is that the requests library behaves the same with `verify=True` and `verify=None`

Reply:
https://github.com/psf/requests/blob/main/requests/sessions.py#L515 `verify=None` is the default value so it is fine and behave the same as verify=True